packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -105,9 +109,20 @@ export default function BackgroundImagePanel( {
 		return {
 			...previousValue,
 			background: {},
+			color: undefined,
 		};
 	}, [] );
 
+	const shouldShowBackgroundColorControls = useHasBackgroundPanel( settings );
```

Comment:
Hmm interesting. Should the color show if there's no background image? This test says "no" since `useHasBackgroundPanel()` checks for `settings?.background?.backgroundImage`.

However, if there's a background image, the color value will have zero effect, unless the user happens to be using a  transparent PNG. 

Also, if a theme turns off background colors in `settings.color.background` should that also have an effect here?

If the intention is to disable background colors then I think the answer is yes. 

It's tricky. Overall, theme.json settings should really be in charge. For example:


```json
"settings": {
		"color": {
			"background": false
		},
		"background": {
			"backgroundImage": false,
		}
}
```

I think the path of least resistance is the following:

1. If `settings.color.background` is `false`, don't show the color picker in the background image control.
2. `settings.background.backgroundImage` is `false` don't show the whole control.

For point 2, when `settings.background.backgroundImage` is `false` it'd be possible to show only the background color control and double up. I think that's a @jasmussen question 😄